### ISSUE TITLE
bluebird unhappy with non-writable 'stack' property

### ISSUE THREAD

**Issue by mvayngrib** [Qix-/node-error-ex#9]
it annoys the bluebird promise library, which complains about promises being rejected with non-errors :)

```js
const ex = require('error-ex')

// from bluebird
const isPropertyWritable = (obj, prop) => {
  var descriptor = Object.getOwnPropertyDescriptor(obj, prop);
  return !!(!descriptor || descriptor.writable || descriptor.set);
}


const CustomError = ex('CustomErr')

console.log(isPropertyWritable(new Error('hey'), 'stack'))
console.log(isPropertyWritable(new CustomError('ho'), 'stack'))
```

**Comment by mvayngrib** [comment]
@Qix- hm, after reviewing #6, i'm not sure this is actually solveable, while preserving your dynamic stack traces

**Comment by Qix-** [comment]
> i'm not sure this is actually solveable, while preserving your dynamic stack traces

It is :) I just check to see if we've overridden the original stack since we (luckily) acquire it each time anyway.

**Comment by Qix-** [comment]
Fix published as `1.3.2`. Thanks again!
